app/json/slate_lineup_configs.json

```diff
@@ -591,28 +524,9 @@
           "2e3ddc90-8def-46d7-b85f-da7525c66fb1",
           "0f322865-64e6-472d-8147-b3d6637a7d67",
           "2f2a3568-901f-4655-8735-daf67e8ecc5d",
-          "505c0126-d54d-42ef-8fe7-0f6a6f24e3a5",
-          "7cb4f497-fd05-42c5-9f78-3650e9ddba21",
-          "e9df8a81-19af-48e2-a90f-05e9a37491ca"
-        ]
-      }
-    ]
-  },
-  {
-    "id": "249850f0-61c0-46f9-a16a-f0553c222800",
-    "description": "Web Home - Fallback Curated",
-    "experiments": [
-      {
-        "description": "default layout",
-        "rankers": [],
-        "slates": [
-          "0737b00e-a21e-4875-a4c7-3e14926d4acf",
-          "2e3ddc90-8def-46d7-b85f-da7525c66fb1",
-          "0f322865-64e6-472d-8147-b3d6637a7d67",
-          "2f2a3568-901f-4655-8735-daf67e8ecc5d",
-          "505c0126-d54d-42ef-8fe7-0f6a6f24e3a5",
+          "b64c873e-7f05-496e-8be4-bfae929c8a04",
           "7cb4f497-fd05-42c5-9f78-3650e9ddba21",
-          "e9df8a81-19af-48e2-a90f-05e9a37491ca"
+          "6d1273a5-055e-4de0-8a5b-5f2b79d37e5c"
```

Comment:
Changed slates to science, food, self-improvement based on [popularity of topic pages](https://app.mode.com/getpocket/reports/8db2aee53ddf).